docs/self-managed/operate-deployment/operate-authentication.md

```diff
@@ -149,7 +149,7 @@ Identity requires the following parameters:
 
 ### Use Identity JWT token to access Operate API
 
-Operate provides a [REST API](/docs/apis-tools/operate-api/operate-api-overview/) under the endpoint `/v1`. Clients can access this API using a JWT access token in an authorization header `Authorization: Bearer <JWT>`.
+Operate provides a [REST API](/apis-tools/operate-api/overview.md) under the endpoint `/v1`. Clients can access this API using a JWT access token in an authorization header `Authorization: Bearer <JWT>`.
```

Comment:
@christinaausley this should fix the first broken link in https://github.com/camunda/camunda-platform-docs/actions/runs/4639190581/jobs/8209771328.

Since there was no `.md` on the end of this path, docusaurus treated it as a browser URL...which it didn't find, because that URL doesn't exist in the 8.1 version yet. 

Instead, I added `.md` to the end so that docusaurus finds it as a source file. I also removed the `/docs` from the beginning because I don't think it's needed, and I personally find it confusing.